pymode/libs/logilab-common-1.4.1/logilab/common/compat.py

```diff
@@ -68,7 +68,10 @@ def method_type(callable, instance, klass):
     reload = reload
 else:
     from io import FileIO, BytesIO, StringIO
-    from imp import reload
+    try:
+        from imp import reload
+    except ImportError:
+        from importlib import reload
```

Comment:
You'll want to do this the other way around; `from imp import reload` will work for now but raise a deprecation error.